packages/cli/src/commands/setup/shadowenv/shadowenv.js

```diff
@@ -0,0 +1,86 @@
+import fs from 'fs'
+import path from 'path'
+
+import boxen from 'boxen'
+import Listr from 'listr'
+
+import { getPaths, writeFile } from '../../../lib'
+import c from '../../../lib/colors'
+
+const shadowenvConfigPath = `${getPaths().base}/.shadowenv.d/rw.lisp`
+
+export const builder = (yargs) => {
+  yargs.option('force', {
+    alias: 'f',
+    default: false,
+    description: 'Overwrite existing configuration',
+    type: 'boolean',
+  })
+}
+
+export const handler = async ({ force }) => {
+  const tasks = new Listr([
+    {
+      title: 'Configuring Shadowenv...',
+      task: (ctx) => {
+        /**
+         * Check if Shadowenv config already exists.
+         * If it exists, throw an error.
+         */
+        if (!force && fs.existsSync(shadowenvConfigPath)) {
+          ctx.error = true
+          throw new Error(
+            'Shadowenv config already exists.\nUse --force to override existing config.'
+          )
+        } else {
+          return writeFile(
+            shadowenvConfigPath,
+            fs
+              .readFileSync(
+                path.resolve(__dirname, 'templates', 'rw.lisp.template')
+              )
+              .toString(),
+            { overwriteExisting: force }
+          )
+        }
+      },
+    },
+    {
+      title: 'Destination:',
+      enabled: (ctx) => !ctx.error,
+      task: (_ctx, task) => {
+        task.title = `  Wrote templates to ${shadowenvConfigPath.replace(
+          getPaths().base,
+          ''
+        )}
+        `
+      },
+    },
+    {
+      title: '',
+      enabled: (ctx) => !ctx.error,
+      task: (_ctx, task) => {
+        task.title = `One more thing...\n\n ${boxen(
+          [
+            c.green('Installation steps for Shadowenv'),
+            '1. On macOS, you can `brew install shadowenv`',
+            '2. In order to avoid an error about an “untrusted” shadowenv, run `shadowenv trust`',
+            '\nSee also: https://shopify.github.io/shadowenv/getting-started/',
```

Comment:
```suggestion
            c.green('Installation steps for Shadowenv'),
            '1. On macOS, you can `brew install shadowenv`',
            '2. To avoid an error about an “untrusted” shadowenv, run `shadowenv trust`',
            "3. Don't forget to configure your shell: https://shopify.github.io/shadowenv/getting-started/#add-to-your-shell-profile",
            '\nSee also: https://shopify.github.io/shadowenv/getting-started/',

```
One thing I had to do besides `brew install shadowenv` was add some config to my shell init files. The wording here makes it sounds like for macOS, I just brew install and I'm good to go.

Reply:
see: https://github.com/redwoodjs/redwood/pull/3499#discussion_r721797065